aci/convert/volume.go

```diff
@@ -82,8 +82,11 @@ func (p projectAciHelper) getAciFileVolumes(ctx context.Context, helper login.St
 func (s serviceConfigAciHelper) getAciFileVolumeMounts(volumesCache map[string]bool) ([]containerinstance.VolumeMount, error) {
 	var aciServiceVolumes []containerinstance.VolumeMount
 	for _, sv := range s.Volumes {
+		if strings.Contains(sv.Source, "/") {
```

Comment:
better rely on sv.Type as https://github.com/compose-spec/compose-go/blob/5afecaa4cb35039bebfc86ae0c3742dab7cae0f3/loader/volume.go#L121 do already handle filepath detection for source.

Reply:
Thx! updated